iface/bind/udp_mux.go

```diff
@@ -282,15 +285,7 @@ func (m *UDPMuxDefault) RemoveConnByUfrag(ufrag string) {
 	for _, c := range removedConns {
 		addresses := c.getAddresses()
 		for _, addr := range addresses {
-			if connList, ok := m.addressMap[addr]; ok {
```

Comment:
out of curiosity how did the update of pion make us remove this code?

Reply:
This code is almost identical with the ice/udp_mux.go file from Pion. I compared the changes and I tried to follow the original pattern.
https://github.com/pion/ice/blob/5139807c1a65b645e515eef009b13193f0aba493/udp_mux.go#L206